### PR TITLE
[RobotSimu] Expose class in python.

### PR DESCRIPTION
**Pull request by florent-lamiraux**

### CODE REVIEW

File: src/tools/robot-simu-python-module-py.cc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#include "dynamic-graph/python/module.hh"
+#include <dynamic-graph/python/signal.hh>
+#include <sot/core/robot-simu.hh>
+#include <sot/core/flags.hh>
+
+namespace dg = dynamicgraph;
+namespace dgs = dynamicgraph::sot;
+
+typedef bp::return_value_policy<bp::reference_existing_object>
+    reference_existing_object;
```
Comment (nim65s) [inline src/tools/robot-simu-python-module-py.cc:11]: What is this for ?

File: src/tools/robot-simu-python-module-py.cc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+BOOST_PYTHON_MODULE(wrap) {
+  bp::import("dynamic_graph.sot.core.wrap");
```
Comment (nim65s) [inline src/tools/robot-simu-python-module-py.cc:14]: Other modules just need `bp::import("dynamic_graph");` ; why is it different here ?

**Review by nim65s — CHANGES_REQUESTED** [review]
Hi @florent-lamiraux,

```python
from dynamic_graph.sot.core.robot_simu import RobotSimu
```

Is already working for me. Could you detail a bit more what is not working for you ? Maybe by adding an unit test showing what you want to achieve ?

This class is not already in https://github.com/stack-of-tasks/sot-core/blob/master/tests/python/python_imports.py, so we should probably start by adding it there, but as this is already working we might want to go further to understand this PR.

Looking at #185, I think just a `self.assertTrue(hasattr(RobotSimu(), 'after'))` would do the job :)
Do you want me to add this ?


On another topic, python modules are build from either a `*-python-module-py.cc` or a `*-python.h`. This entity was using `robot-simu-python.h` before, so if you add `robot-simu-python-module-py.cc`, you should remove the header.

Ref: https://github.com/stack-of-tasks/sot-core/blob/aba69263f2653920daeb00efcefff50d44b30c93/src/CMakeLists.txt#L112

File: src/tools/robot-simu-python-module-py.cc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#include <boost/python.hpp>
+#include "dynamic-graph/python/module.hh"
+#include <dynamic-graph/python/signal.hh>
+#include <sot/core/robot-simu.hh>
+#include <sot/core/flags.hh>
```
Comment (nim65s) [inline src/tools/robot-simu-python-module-py.cc:5]: I think we only need `dynamic-graph/python/module.hh` & `sot/core/robot-simu.hh`

File: src/tools/robot-simu-python-module-py.cc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+namespace dg = dynamicgraph;
+namespace dgs = dynamicgraph::sot;
+
+typedef bp::return_value_policy<bp::reference_existing_object>
+    reference_existing_object;
+
+BOOST_PYTHON_MODULE(wrap) {
+  bp::import("dynamic_graph.sot.core.wrap");
+  dynamicgraph::python::exposeEntity<dgs::RobotSimu,
+				     bp::bases<dg::sot::Device> >();
```
Comment (nim65s) [inline src/tools/robot-simu-python-module-py.cc:16]:
```suggestion
  dg::python::exposeEntity<dgs::RobotSimu, bp::bases<dg::sot::Device> >();
```

**Comment by florent-lamiraux** [comment]
@nim65s : thank you for your feedback. After reinstalling the package, I am not able to reproduce the issue.
I will therefore close this pull request.

**Pull request closed without merging**